applications/migrations/0060_change_application_status_values.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.13 on 2022-06-15 03:17
+
+from django.db import migrations
+
+from applications.models import ApplicationStatus
+
+
+def change_status_to_new(*args, **kwargs):
+    ApplicationStatus.objects.filter(status="declined").update(status="handled")
+
+
+# Rolling back to original values is not possible because
+# we can't assume that all 'handled' applications are 'declined'
+def change_status_to_old(*args, **kwargs):
+    pass
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("applications", "0059_change_application_round_status_values"),
+    ]
+
+    operations = [migrations.RunPython(change_status_to_new, change_status_to_old)]
```

Comment:
Nice that there's a reverse code :+1:  
Tip wednesday: instead of giving none doing function, we can give/say `migrations.RynPython.noop`:  `operations = migrations.RunPython(change_status_to_new, migrations.RunPython.noop)]` so it does nothing but can be "reverted"

Reply:
Lovely! I cleaned the scripts 🧹